src/SearchField/SearchFieldSubmitButton.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react';
+
+import { SearchFieldContext } from './SearchFieldAdvanced';
+
+const SearchFieldSubmitButton = () => {
+  const { screenReaderText, icons, refs } = useContext(SearchFieldContext);
+
+  return (
+    <button
+      type="submit"
+      className="btn px-2"
```

Comment:
I'm mildly concerned about using `px-2` and other padding/margin utilities, but don't have a better solution in mind. Utility classes are annoying to override so if we're putting them in we should be confident that they are not something that should be themable. In this case maybe that's true.

Reply:
Fair point. In this case, I think that's largely true. However, could also move the use of the utility classes like this one to the custom SCSS file for the component with `@extend .px-2`.

Reply:
I think it's fine here. If it gets in the way we can chagne in the future. Wouldn't be a breaking change I think